### PR TITLE
ES6

### PR DESCRIPTION
**Pull request by lucasmazza**
* [x] Covert `page.js` to ES6
* [x] Build usable assets
* [x] Review `README.md`
* [x] Attempt to reuse ES6 code on the test suite or tooling scripts.
* [x] ???
* [x] PROFIT.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "page",
   "main": [
-    "./page.js",
-    "./page.min.js"
+    "./dist/page.js",
```
Comment (rafaelfranca) [inline bower.json]: Does this file still exists?

File: src/page.es6 [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-  var Page = function() {
-    this.initializers = {};
-    this.initializers[':before'] = [];
-    this.initializers[':after'] = [];
-  };
+// Internal: Initializes a 'Page' object.
+// Browser environments does not need to call this constructor
+// since a 'page' object will be exported in the global namespace.
+class Page {
+  static variantsDelimiter() { return '+'; }
```
Comment (rafaelrinaldi) [inline src/page.es6]: You could use a getter here.

File: src/page.es6 [hunk 3]
```diff
@@ -127,22 +126,24 @@
...
     return true;
-  };
+  }
+}
 
-  root.page = new Page();
-  root.Page = Page;
-})(this);
+export default Page;
+if(window) {
```
Comment (rafaelrinaldi) [inline src/page.es6]: I believe you don't need to do that. The transpiler should handle a proper export strategy.

Comment (lucasmazza) [inline src/page.es6]: It does by defining the global `Page` object, but the `page` instance (a syntax sugar that we have here) needs to be handled manually.

File: src/page.es6 (hunk 2 above)
Comment (lucasmazza) [inline src/page.es6]: I wanted to mimic a class level constant here (previously this was a private variable) but even which the `static` here this still can be mutated by the outside world.

**Pull request merged**